akka-docs/src/main/paradox/typed/distributed-pub-sub.md

```diff
@@ -0,0 +1,204 @@
+# Distributed Publish Subscribe in Cluster
+
+@@@ note
+For the Akka Classic documentation of this feature see @ref:[Classic Distributed Publish Subscribe](../distributed-pub-sub.md).
+@@@
+
+@@project-info{ projectId="akka-cluster-typed" }
+
+## Dependency
+
+To use Distributed Publish Subscribe add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+  group=com.typesafe.akka
+  artifact=akka-cluster-typed_$scala.binary_version$
```

Comment:
it will also `require akka-cluster-tools`, where the Classic PubSub is located

Reply:
Added